src/app/business/routes/dataset/components/detail/components/tabs/index.js

```diff
@@ -45,11 +52,26 @@ descLoading, item, tabIndex, openerLoading, setTabIndex, addNotification,
         </TabList>
         <TabPanel>
             {descLoading && <PulseLoader size={6} />}
-            {!descLoading && <Description item={item} />}
+            {!descLoading && descForbidden && (
+                <ForbiddenDescription
+                    model="dataset"
+                    permissionsTabIndex={4}
+                    setTabIndex={setTabIndex}
+                />
+            )}
+            {!descLoading && !descForbidden && <Description item={item} />}
         </TabPanel>
         <TabPanel>
             {openerLoading && <PulseLoader size={6} />}
-            {!openerLoading && item && item.opener && item.opener.content && (
+            {!openerLoading && openerForbidden && (
+                <Fragment>
```

Comment:
Should it be in its own component as `ForbiddenDescription` is?
Should we create a more generic `Fordidden` component?

Reply:
This makes a lot of sense, I'll update it!

Reply:
Done!